app/views.py

```diff
@@ -3,13 +3,10 @@
 import re
 from json import loads
 
-from sqlalchemy import func, update
```

Comment:
Another issue that will be caught by a linter is unnecessary imports; don't import any symbols that you don't end up using.